cypress/integration/pages/organizations.spec.js

```diff
@@ -47,7 +47,7 @@ describe('Contributors Page (using API)', () => {
     cy.intercept(`${Cypress.env('REACT_APP_API_URL')}/api/organizations/`).as(
       'getOrganizations'
     );
-    cy.visit('/organizations/affiliated');
+    cy.visit('/organizations/all');
```

Comment:
Is there a reason to change this route to /all ?

Reply:
The test is failing if it is affiliated. To pass the test I gave this